index.html

```diff
@@ -191,7 +191,7 @@ <h2>Languages</h2>
                             <img src="images/Icons/icon-frameworks.svg" alt="icon-languages">
                         </div>
                         <div class="about-datails">
-                            <h2>Frameworks</h2>
+                            <h2>frameworks</h2>
```

Comment:
Please try to make your title start with a capital letter as your Figma design.😒😒